### PR TITLE
fix: build tailwind with rollup in TS

### PR DESCRIPTION
**Pull request by ahmaxed**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

Should be ready to go in.

Here is the screen shot of the imported Alert from the bundle. (there style overlaps, but it works for now)

Screen Shot 2022-01-08 at 2 06 44 PM

### CODE REVIEW

File: tools/ui-components/src/index.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
 // Use this file as the entry point for component export
+import './global.css';
```
Comment (ojeytonwilliams) [inline tools/ui-components/src/index.ts:2]: @ahmadabdolsaheb was this the trick we were missing?

Comment (ahmadabdolsaheb) [reply tools/ui-components/src/index.ts:2]: no in the tailwind config, I did not account for TSX files.

File: tools/ui-components/package.json [hunk 2]
```diff
@@ -45,6 +46,7 @@
     "babel-loader": "8.2.3",
     "babel-plugin-transform-react-remove-prop-types": "0.4.24",
     "cross-env": "7.0.3",
+    "cssnano": "^5.0.14",
```
Comment (huyenltnguyen) [inline tools/ui-components/package.json:49]: I think we need `postcss-cli` and some postcss config update in order for this to work? Ref: https://cssnano.co/docs/getting-started/#using-postcss-cli

Though I'm not really sure here to be honest (I'm not familiar with the plugin and still catching up 😅)

Comment (ahmadabdolsaheb) [reply tools/ui-components/package.json:49]: Right. The postcss config has to change, I will take it out for now.

Comment (ahmadabdolsaheb) [inline tools/ui-components/package.json:49]:
```suggestion
```

**Review by huyenltnguyen — DISMISSED** [review]
LGTM. Thanks @ahmadabdolsaheb!

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

**Pull request merged**